### PR TITLE
Color updates for icons relative to the 1.5 -> 1.6 design tokens upgrade

### PR DESCRIPTION
**Pull request by georgewrmarshall**
**Description**

We updated the design-tokens from 1.5 -> 1.6 which included some color updates. This PR updates some of the relative components to follow the correct color patterns outlined in the design system.

**Checklist**

* [x] ~There is a related GitHub issue~ NA
* [x] ~Tests are included if applicable~ NA
* [x] ~Any added code is fully documented~ NA

**Screenshots/Recordings**

Because the changes touch a lot of files I will add screenshots as comments under the relative code

### CODE REVIEW

File: app/components/Base/TabBar.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ function TabBar({ ...props }) {
     <DefaultTabBar
       underlineStyle={styles.tabUnderlineStyle}
       activeTextColor={colors.primary.default}
-      inactiveTextColor={colors.text.muted}
+      inactiveTextColor={colors.text.alternative}
       backgroundColor={colors.background.default}
       tabStyle={styles.tabStyle}
       textStyle={styles.textStyle}
```
Comment (georgewrmarshall) [inline app/components/Base/TabBar.js:36]: Making the text of the tab bar more readable. text.muted should be assumed to be non-accessible and usually used for disabled/inactive components. This use case the tab can still be clickable.

### Before
Screen Shot 2022-05-26 at 4 08 49 PM


### After
Screen Shot 2022-05-26 at 4 00 23 PM

File: app/components/UI/AccountList/AccountElement/index.js [hunk 2]
```diff
@@ -75,7 +75,7 @@ const createStyles = (colors) =>
       paddingVertical: 3,
       borderRadius: 10,
       borderWidth: 1,
-      borderColor: colors.icon.default,
+      borderColor: colors.border.default,
     },
   });
```
Comment (georgewrmarshall) [inline app/components/UI/AccountList/AccountElement/index.js:81]: Should use border color for borders

### Before
Screen Shot 2022-05-26 at 4 11 33 PM

### After

Screen Shot 2022-05-26 at 4 12 14 PM

File: app/components/UI/AssetElement/index.js [hunk 3]
```diff
@@ -72,7 +72,7 @@ export default class AssetElement extends PureComponent {
           <Icon
             name="ios-arrow-forward"
             size={24}
-            color={colors.icon.muted}
+            color={colors.icon.alternative}
             style={styles.arrowIcon}
           />
         </View>
```
Comment (georgewrmarshall) [inline app/components/UI/AssetElement/index.js:78]: Accompanying interactive icons should use `icon.alternative`

### Before

Screen Shot 2022-05-26 at 8 52 26 AM

### After

Screen Shot 2022-05-26 at 3 35 00 PM

File: app/components/UI/CollectibleContracts/index.js [hunk 4]
```diff
@@ -69,7 +69,7 @@ const createStyles = (colors) =>
       height: 76,
       marginTop: 30,
       marginBottom: 12,
-      tintColor: colors.icon.default,
+      tintColor: colors.icon.muted,
     },
     emptyTitleText: {
       fontSize: 24,
```
Comment (georgewrmarshall) [inline app/components/UI/CollectibleContracts/index.js:75]: Reducing contrast of supplementary icon

### Before
Screen Shot 2022-05-25 at 3 50 32 PM

### After
Screen Shot 2022-05-25 at 3 50 15 PM

File: app/components/UI/DrawerView/index.js [hunk 5]
```diff
@@ -874,22 +874,32 @@ class DrawerView extends PureComponent {
...
 
     return (
-      <MaterialIcon name={name} size={size || 24} color={colors.icon.default} />
+      <MaterialIcon
+        name={name}
+        size={size || 24}
+        color={colors.icon.alternative}
+      />
     );
   }
```
Comment (georgewrmarshall) [inline app/components/UI/DrawerView/index.js:905]: Additional icons for interactive elements should use `icon.alternative`

### Before
Screen Shot 2022-05-25 at 4 25 20 PM

### After
Screen Shot 2022-05-26 at 3 33 09 PM

File: app/components/UI/FiatOnRampAggregator/components/Quote.tsx [hunk 6]
```diff
@@ -50,7 +50,7 @@ const createStyles = (colors: Colors) =>
     },
     infoIcon: {
       marginLeft: 8,
-      color: colors.icon.default,
+      color: colors.icon.alternative,
     },
     data: {
       overflow: 'hidden',
```
Comment (georgewrmarshall) [inline app/components/UI/FiatOnRampAggregator/components/Quote.tsx:56]: Additional interactive icons for not primary actions should use icon.alternative. Don't have a screenshot but it's an (i) info icon

File: app/components/UI/FiatOnRampAggregator/components/RegionModal.tsx [hunk 7]
```diff
@@ -304,7 +304,7 @@ const RegionModal: React.FC<Props> = ({
                         name="ios-close-circle"
                         size={20}
                         style={styles.searchIcon}
-                        color={colors.icon.default}
+                        color={colors.icon.alternative}
                       />
                     </TouchableOpacity>
                   )}
```
Comment (georgewrmarshall) [inline app/components/UI/FiatOnRampAggregator/components/RegionModal.tsx:310]: icon.default should usually be reserved for standalone icons (x)close (<)back

### Before
Screen Shot 2022-05-25 at 3 54 55 PM

### After
Screen Shot 2022-05-25 at 3 54 39 PM

File: app/components/UI/FiatOnRampAggregator/components/TokenSelectModal.tsx [hunk 8]
```diff
@@ -54,7 +54,7 @@ const createStyles = (colors: Colors) =>
     },
     searchIcon: {
       marginHorizontal: 8,
-      color: colors.icon.default,
+      color: colors.icon.alternative,
     },
     input: {
       ...fontStyles.normal,
```
Comment (georgewrmarshall) [inline app/components/UI/FiatOnRampAggregator/components/TokenSelectModal.tsx:60]: Same as above

File: app/components/UI/NetworkList/index.js [hunk 9]
```diff
@@ -324,7 +324,7 @@ export class NetworkList extends PureComponent {
     const styles = this.getStyles();
     const isMainnet =
       provider.type === MAINNET ? (
-        <Icon name="check" size={15} color={colors.icon.default} />
+        <Icon name="check" size={16} color={colors.success.default} />
       ) : null;
     const { color: mainnetColor, name: mainnetName } = Networks.mainnet;
```
Comment (georgewrmarshall) [inline app/components/UI/NetworkList/index.js:330]: Updates to check color and making them all the same size

### Before
Screen Shot 2022-05-26 at 4 34 59 PMScreen Shot 2022-05-26 at 4 35 12 PM


### After
Screen Shot 2022-05-25 at 3 58 42 PMScreen Shot 2022-05-25 at 3 58 48 PM

File: app/components/UI/PaymentRequest/index.js [hunk 10]
```diff
@@ -782,7 +782,7 @@ class PaymentRequest extends PureComponent {
                     <FontAwesome
                       name="exchange"
                       size={18}
-                      color={colors.icon.default}
+                      color={colors.primary.default}
                       style={{ transform: [{ rotate: '270deg' }] }}
                     />
                   </TouchableOpacity>
```
Comment (georgewrmarshall) [inline app/components/UI/PaymentRequest/index.js:788]: Making exchange icon look interactive

###Before
Screen Shot 2022-05-25 at 4 00 48 PM

### After
Screen Shot 2022-05-25 at 4 04 02 PM

File: app/components/UI/ReceiveRequest/index.js [hunk 11]
```diff
@@ -311,7 +313,7 @@ class ReceiveRequest extends PureComponent {
               <EvilIcons
                 name={Device.isIos() ? 'share-apple' : 'share-google'}
                 size={25}
-                color={colors.icon.default}
+                color={colors.primary.default}
               />
             </TouchableOpacity>
           </TouchableOpacity>
```
Comment (georgewrmarshall) [inline app/components/UI/ReceiveRequest/index.js:319]: Updating copy button

### Before
Screen Shot 2022-05-25 at 4 22 40 PM

### After
Screen Shot 2022-05-25 at 4 22 16 PM

File: app/components/UI/SettingsDrawer/index.js [hunk 12]
```diff
@@ -39,7 +39,7 @@ const createStyles = (colors) =>
     },
     icon: {
       bottom: 8,
-      color: colors.icon.muted,
+      color: colors.icon.alternative,
       left: 4,
       position: 'relative',
     },
```
Comment (georgewrmarshall) [inline app/components/UI/SettingsDrawer/index.js:45]: Additional icons for interactive elements should use `icon.alternative`

### Before
Screen Shot 2022-05-25 at 4 25 20 PM

### After
Screen Shot 2022-05-26 at 3 33 09 PM

File: app/components/UI/Swaps/QuotesView.js [hunk 13]
```diff
@@ -1975,7 +1974,10 @@ function SwapsQuotesView({
...
                       fee: selectedQuote.fee,
                     })} `}
-                    <FAIcon name="info-circle" style={styles.infoIcon} />
+                    <MaterialCommunityIcons
+                      name="information"
+                      style={styles.infoIcon}
+                    />
                   </Text>
                 </TouchableOpacity>
               </View>
```
Comment (georgewrmarshall) [inline app/components/UI/Swaps/QuotesView.js:1983]: Updating info icons

### Before

Screen Shot 2022-05-25 at 4 08 43 PM

### After

Screen Shot 2022-05-25 at 4 08 09 PM

File: app/components/UI/Swaps/components/TokenSelectModal.js [hunk 14]
```diff
@@ -69,7 +69,7 @@ const createStyles = (colors) =>
     },
     searchIcon: {
       marginHorizontal: 8,
-      color: colors.icon.default,
+      color: colors.icon.alternative,
     },
     input: {
       ...fontStyles.normal,
```
Comment (georgewrmarshall) [inline app/components/UI/Swaps/components/TokenSelectModal.js:75]: Updating search icon in text input

File: app/components/Views/ImportPrivateKey/index.js [hunk 15]
```diff
@@ -75,7 +75,7 @@ const createStyles = (colors) =>
       fontSize: 50,
       marginTop: 0,
       marginLeft: 0,
-      color: colors.text.default,
+      color: colors.icon.alternative,
     },
     buttonWrapper: {
       flex: 1,
```
Comment (georgewrmarshall) [inline app/components/Views/ImportPrivateKey/index.js:81]: Updating import icon to be less contrasting

### Before
Screen Shot 2022-05-25 at 4 13 49 PM


### After
Screen Shot 2022-05-25 at 4 13 36 PM

File: app/components/Views/SendFlow/AddressInputs/index.js [hunk 16]
```diff
@@ -584,7 +578,7 @@ export const AddressFrom = (props) => {
                 size={20}
                 style={[
                   styles.dropdownIcon,
-                  highlighted ? styles.iconHighlighted : styles.iconOpaque,
+                  highlighted ? styles.iconHighlighted : styles.inputIconOpaque,
                 ]}
               />
             </View>
```
Comment (georgewrmarshall) [inline app/components/Views/SendFlow/AddressInputs/index.js:584]: Updating input icons

Screen Shot 2022-05-26 at 3 45 04 PM
Screen Shot 2022-05-26 at 3 45 10 PM

File: app/components/Views/Settings/Contacts/ContactForm/index.js [hunk 17]
```diff
@@ -367,7 +367,7 @@ class ContactForm extends PureComponent {
                   <AntIcon
                     name="scan1"
                     size={20}
-                    color={colors.icon.default}
+                    color={colors.primary.default}
                     style={styles.scanIcon}
                   />
                 </TouchableOpacity>
```
Comment (georgewrmarshall) [inline app/components/Views/Settings/Contacts/ContactForm/index.js:373]: Using action colors to highlight unfamiliar actions other than close

### Before
Screen Shot 2022-05-26 at 4 47 08 PM

### After
Screen Shot 2022-05-26 at 4 47 25 PM

**Review by Cal-L — APPROVED** [review]
LGTM

**Pull request merged**